locales/en.default.json

```diff
@@ -359,6 +359,11 @@
       "remove_all_items_confirmation": "Remove all {{ quantity }} items from your cart?",
       "remove_all": "Remove all",
       "cancel": "Cancel"
+    },
+    "quick_order_form": {
+      "search": "Search by product, variant or SKU",
+      "search_number_items": "{{ count }} items",
+      "line_total": "Line total"
```

Comment:
@eugenekasimov  `{{ count }} items` already exists in the Dawn repo 👀 can we refer to that line to avoid duplicating translations. Also a note that translations arent approved so lets hold off on requesting them :)

Reply:
Honestly, I agree with the idea of not duplicating the same translation. However, I'm not a big fan of using translations related to different sections, even if they are the same. As you mentioned, we already have `{{ count }} items` and it's living under the header section.

I'm okay with using the existing translation, but I just want to make sure that we are comfortable with that. Thoughts? 

cc: @melissaperreault @andrewetchen

Reply:
I agree it doesnt make sense since its under header. However, we should avoid creating duplicate translations. Can we move it from the `header` to `general`?

```
      "cart_count": {
        "one": "{{ count }} item",
        "other": "{{ count }} items"
      }
```

Reply:
Since we changed "Product subtotal" to "Subtotal" in the total bar, I'm going to use the existing "Subtotal" from {{ 'customer.order.subtotal' | t }}. This makes sense to me because, from a certain perspective, the total bar of the quick order form reflects the customer's order.

Unfortunately, I cannot say the same about {{ count }} items. The existing string is located under the header > cart_count. So the whole path looks like {{ 'sections.header.cart_count' | t }}, and even if we move it under the general section, it will still have a reference to "cart_count". This is confusing because we are going to use it for the quick order form search, and in our case, it will represent the quantity of items we render in the quick order form. What I'm trying to say is, if we want to reuse the existing {{ count }} items, I would move only "other": "{{ count }} items" to the general section, but not the whole "cart_count": { }.

It will look something like 

```
"general": {
   ...
   "other" : {
      "count_items" : "{{ count }} items"
    }
}

```

Reply:
I think there are two things here 👀 

1. I dont think customers makes the most sense since it's a separate section and you have to be logged in. 🤔 I think in that case it also makes sense to move it.
2. I would move `"count_items" : "{{ count }} items"` to a general section in the translations and remove it from the header. This way we can use it for both the header and this section.

I think this is something we may run into again :) But I think the general rule is to
1. Avoid duplication
2. Move it to a general translation object if the string is reused somewhere else

We can take this to the channel to get other developers povs too